packages/nimble-components/src/table-column/date-text/cell-view/index.ts

```diff
@@ -20,24 +21,80 @@ export class TableColumnDateTextCellView extends TableColumnTextCellViewBase<
 TableColumnDateTextCellRecord,
 TableColumnDateTextColumnConfig
 > {
-    private static readonly formatter = new Intl.DateTimeFormat(undefined, {
-        dateStyle: 'medium',
-        timeStyle: 'medium'
-    });
+    private static formatter: Intl.DateTimeFormat;
```

Comment:
Of course. Duh. I'm not sure it's worth maintaining a map that stores one formatter per column, so I'm just creating one for every cell view and group header view.

Reply:
I'm curious to hear your rationale for it not being appropriate to put the formatter in the columnConfig.

Reply:
I feel like @rajsite made a point of it being just for "dumb" configuration data, and a formatter object is instead something functional created using that configuration. To be honest, since we're not serializing the columnConfig, I'm not sure what practical reasons there would be for enforcing that restriction. It certainly would be convenient to share the formatter via columnConfig.

Reply:
Talked to Milan, and he clarified his definition of "dumb" config data. Basically he meant state that was independent and not mutable by/exposed to other parts of the system. So the formatter fits that definition and can reasonably be stored in the columnConfig. 😊